### PR TITLE
change affects to involves for events

### PR DESCRIPTION
**Pull request by fosterlynn**
I think we agree on this, so I thought I'd get that pushed.

### CODE REVIEW

File: release-doc-in-process/all_vf.TTL [hunk 1]
```diff
@@ -178,10 +178,10 @@ vf:recipeOutputOf
 vf:involves
         a                   owl:ObjectProperty ;
         rdfs:label          "involves" ;
-        rdfs:domain         [ owl:unionOf (vf:Commitment vf:Intent) ] ; 
+        rdfs:domain         [ owl:unionOf (vf:Commitment vf:Intent vf:EconomicEvent) ] ; 
         rdfs:range          vf:EconomicResource ;
         vs:term_status      "unstable" ;
-        rdfs:comment        "Economic resource the commitment commits or the intent intends to affect. Could be an actual resource or a classification / specification of resource." .
+        rdfs:comment        "Economic resource referenced by the flow." .
```
Comment (elf-pavlik) [inline release-doc-in-process/all_vf.TTL]: What do you think about: *Economic resource involved in the flow*

File: release-doc-in-process/all_vf.TTL [hunk 2]
```diff
@@ -198,13 +198,13 @@ vf:relationship  a          owl:ObjectProperty ;
...
-        rdfs:range          vf:EconomicResource ;
-        vs:term_status      "unstable" ;
-        rdfs:comment        "The economic resource that is decremented or incremented by the economic event, or just what the economic event is about." .
+#vf:affects
+#        a                   owl:ObjectProperty ;
+#        rdfs:label          "affects" ;
+#        rdfs:domain         vf:EconomicEvent ;
+#        rdfs:range          vf:EconomicResource ;
+#        vs:term_status      "unstable" ;
+#        rdfs:comment        "The economic resource that is decremented or incremented by the economic event, or just what the economic event is about." .
```
Comment (elf-pavlik) [inline release-doc-in-process/all_vf.TTL]: Let's just delete it, we can always find it in git history so we have 'undo' button there if needed.

File: release-doc-in-process/all_vf.TTL (hunk 1 above)
Comment (fosterlynn) [reply release-doc-in-process/all_vf.TTL]: I try not to repeat the word, but no problem with me - changed.

File: release-doc-in-process/all_vf.TTL (hunk 2 above)
Comment (fosterlynn) [reply release-doc-in-process/all_vf.TTL]: done

**Pull request merged**